lib/views/pages/home/home_page.dart

```diff
@@ -22,14 +23,21 @@ class HomePage extends HookConsumerWidget {
       }
```

Comment:
This should be navigateToLoginPageOnLogout(context, ref); It should moreover not be duplicated with app_top_bar.dart:23. I'd recommend to only keep the one here (calling navigateToLoginPageOnLogout naturally) and to remove the one of app_top_bar.dart

Reply:
Good catch, makes also sense to keep the one in the home page as the logout button in the app bar is temporary.

Reply:
No, I don't think we would want it in the app top bar, since it only needs to be called once in the entire navigation stack. It's nice you also removed it in your commit :)